### PR TITLE
Replace obsolete ksonnet tests with Tekton.

### PR DESCRIPTION
**Pull request by jlewi**
* This only runs the go unittests not the E2E tests.

Related to #247

### CODE REVIEW

File: test/pipeline-runs/go-tests-run.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Pipelinerun for go unittests
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineRun
+metadata:
+  generateName: go-test-
+  namespace: kf-ci
+  labels:
+    pipeline: go-test
+    repo: gcp-blueprints
```
Comment (Jeffwan) [inline test/pipeline-runs/go-tests-run.yaml:9]: This is a separate repo? Seems it's unnecessary?

File: test/pipeline-runs/go-tests-run.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+spec:
+  params:
+  # When running under prow this value will automatically be replaced 
+  # by the appropriate GCS location for the test.
+  # So the value here only matters for manual runs.
+  - name: artifacts-gcs
+    value: "gs://kubeflow-ci-deployment/gabrielwen-testing-2"
```
Comment (Jeffwan) [inline test/pipeline-runs/go-tests-run.yaml:16]: minor:  Is there a more generic name can be used?

**Review by Jeffwan — APPROVED** [review]

**Comment by Jeffwan** [comment]
I am working on ML tracking related stories. I will help put some time helping review the code changes

File: test/pipeline-runs/go-tests-run.yaml (hunk 2 above)
Comment (jlewi) [reply test/pipeline-runs/go-tests-run.yaml]: I changed it to an empty place holder.

File: test/pipeline-runs/go-tests-run.yaml (hunk 1 above)
Comment (jlewi) [reply test/pipeline-runs/go-tests-run.yaml]: I fixed it. The labels make it easy to find all pipelines for a given repo/test

**Comment by jlewi** [comment]
Thanks @Jeffwan; PTAL

**Review by Jeffwan — APPROVED** [review]

**Comment by Jeffwan** [comment]
/lgtm

If OWNERS are not active, we can manually `squash and merge` PR.

**Pull request merged**